mongodb/src/main/java/com/yahoo/ycsb/db/AsyncMongoDbClient.java

```diff
@@ -0,0 +1,477 @@
+/**                                                                                                                                                                                
+ * Copyright (c) 2013-2014, Allanbank Consulting, Inc. All rights reserved.                                                                                                                             
```

Comment:
I will update the license.  Is there a assignment document that need to be submitted?